config/0chain_blobber.yaml

```diff
@@ -67,6 +67,8 @@ min_confirmation: 50
 
 block_worker: http://198.18.0.98:9091
 
+challenge_completion_time: 2m
```

Comment:
3m

Reply:
fixed

Reply:
its now 30s @peterlimg on dev networks

Reply:
don't worry about it. it is configurable on yml file.